### DISCUSSION TITLE
Add the windows 11 logo to the os logos

### DISCUSSION THREAD

**Discussion by tuklu** [ryanoasis/nerd-fonts#1500] [Ideas]
Add the Windows 11 logo to the logo directory so that the Windows 11 terminal displays the 11 logo.

**Comment by Finii** [comment]
Hmm, I thought we have the current Windows logo at `E62A` 🤔 

<img width="947" alt="Screenshot 2024-01-29 at 23 30 40" src="https://github.com/ryanoasis/nerd-fonts/assets/16012374/524accca-4e2a-4d8b-9b8c-b7021d3a8a8f">
